### PR TITLE
Add UTs for sequencer package 

### PR DESCRIPTION
**Pull request by begmaroman**

### CODE REVIEW

File: sequencer/tracker_test.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+				require.Error(t, err)
+				require.EqualError(t, tt.err, err.Error())
+			} else {
+				require.NoError(t, err)
+			}
+		})
+	}
+}
+
+func TestTracker(t *testing.T) {
```
Comment (Stefan-Ethernal) [inline sequencer/tracker_test.go:89]: Is this some kind of BDD test? If yes, then wdyt to include some prefix to its name (e.g. `TestBDD_Tracker`)?

File: go.mod [hunk 2]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/lib/pq v1.10.7
 	github.com/miguelmota/go-solidity-sha3 v0.1.1
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/onsi/gomega v1.30.0
```
Comment (Stefan-Ethernal) [inline go.mod:17]: By the way not sure whether we needed to introduce that dependency. Based on how it is used, it shouldn't be too much of an effort to implement a function that would periodically execute some custom logic and within its body, we could make assertions using `testify` lib and make certain assertions. WDYT?

File: sequencer/call_test.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+			}))
+			defer svr.Close()
+
+			got, err := GetData(svr.URL, tt.batchNum)
+			if tt.err != nil {
+				require.Error(t, err)
+				require.EqualError(t, tt.err, err.Error())
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, *tt.expectedData, *got)
```
Comment (Stefan-Ethernal) [inline sequencer/call_test.go:91]: Nitpick, I think there is no need to de-reference the pointers, but we can compare two objects this way as well.
```suggestion
				require.Equal(t, tt.expectedData, got)
```

**Review by Stefan-Ethernal — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (begmaroman) [reply go.mod]: Done

File: sequencer/call_test.go (hunk 3 above)
Comment (begmaroman) [reply sequencer/call_test.go]: Done

**Pull request merged**